documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -762,7 +762,7 @@ you must set one of the following options:
 
 ==== Ad hoc read-only incremental snapshots
 
-When the MySQL connection is read-only, you can use the {link-prefix}:{link-signalling}#debezium-signaling-enabling-kafka-signaling-channel[kafka signaling channel] to send messages to Kafka topic.
+When the MySQL connection is read-only, you can use any of the {link-prefix}:{link-signalling}#overview[available signaling channels] without the requirement to use the `source` channel.
```

Comment:
```suggestion
When the MySQL connection is read-only, you can use any of the {link-prefix}:{link-signalling}#sending-signals-to-a-debezium-connector[available signaling channels] without the requirement to use the `source` channel.
```
To enable this link to work downstream, we can't use the `overview` anchor, because it's in a section that's conditionalized as community content.